tests/pkcs11_test.c

```diff
@@ -3,13 +3,59 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
+/**
```

Comment:
Do you think PKCS11.md in the root or something might be a good place for this? Especially so we can link to it from downstream packages?

Reply:
Great idea. Doing it.